Extension/ui/settings.html

```diff
@@ -498,18 +498,20 @@
     <div class="section">
         <div class="section-title">IntelliSense mode</div>
         <div class="section-text">
-            The IntelliSense mode used by the IntelliSense engine. <code>msvc-x64</code> maps to Visual Studio mode with 64-bit pointer sizes. 
-            <code>clang-x64</code> maps to CLang mode with 64-bit pointer sizes. <code>gcc-x64</code> maps to GCC mode with 64-bit pointer sizes. 
-            The <code>${default}</code> mode will choose the default for that platform. 
+            The IntelliSense mode to use that maps to an architecture-specific variant of MSVC, gcc, or Clang. 
+            If not set or if set to ${default}, the extension will choose the default for that platform. 

```

Comment:
Add `<code>` style around `${default}` like below:

`<code>${default}</code>`

Reply:
Thank you